### PR TITLE
Register route and some callbacks

### PR DESCRIPTION
**Pull request by marco-c**
There's still an error in the test that I need to figure out.

### CODE REVIEW

**Comment by marco-c** [comment]
Looks like `server` is null in the test, @danielbachhuber any idea why?

File: lib/class-wp-rest-plugins-controller.php [hunk 1]
```diff
@@ -12,7 +12,28 @@ public function __construct() {
...
+			array(
+				'methods'         => WP_REST_Server::READABLE,
+				'callback'        => array( $this, 'get_items' ),
+				'permission_callback' => array( $this, 'get_items_permissions_check' ),
+				'args'            => $this->get_collection_params(),
+			),
+			'schema' => array( $this, 'get_item_schema' ),
+		) );
+
+		register_rest_route( $this->namespace, '/' . $this->rest_base . '/(?P<id>[\d]+)', array(
```
Comment (danielbachhuber) [inline lib/class-wp-rest-plugins-controller.php]: Do plugins have ids?

Comment (marco-c) [inline lib/class-wp-rest-plugins-controller.php]: Just a placeholder for now

Comment (marco-c) [inline lib/class-wp-rest-plugins-controller.php]: We'll probably use the slug

**Pull request merged**